lib/susedistribution.pm

```diff
@@ -198,6 +198,7 @@ sub become_root {
     my ($self) = @_;
 
     $self->script_sudo('bash', 0);
+    wait_still_screen;
```

Comment:
I though I have seen the root there, but it's from something else ...
The problem is not the speed of typing but when we start to type, I guess.